middleware/favicon/README.md

```diff
@@ -67,6 +72,7 @@ type Config struct {
 ```go
 var ConfigDefault = Config{
 	Next: nil,
-	File:	""
+	File:	"",
+	URL: "/favicon.ico"
```

Comment:
This wouldn't compile without a slash at the end of line here.

Reply:
Also, please properly align. Also the `File` line seems to include a redundant space.